### ISSUE TITLE
Broken failures

### ISSUE THREAD

**Issue by sergeyshaykhullin** [benmccallum/fairybread#28]
Failures are broken

```json
"extensions": {
    "message": "Validation failed: \r\n -- Text: Text should be not empty",
    "stackTrace": "   at FairyBread.InputValidationMiddleware.OnInvalid(IMiddlewareContext context, IEnumerable`1 invalidValidationResults)\r\n   at FairyBread.InputValidationMiddleware.InvokeAsync(IMiddlewareContext context)\r\n   at HotChocolate.Utilities.MiddlewareCompiler`1.ExpressionHelper.AwaitTaskHelper(Task task)\r\n   at HotChocolate.Execution.Processing.ResolverTask.ExecuteResolverPipelineAsync(CancellationToken cancellationToken)\r\n   at HotChocolate.Execution.Processing.ResolverTask.TryExecuteAsync(CancellationToken cancellationToken)",
    "code": "FairyBread_ValidationError",
    "Failures": "System.Linq.Enumerable+SelectEnumerableIterator`2[FluentValidation.Results.ValidationFailure,System.Object]"
}
```

Failures should be array of validation failure

**Comment by benmccallum** [comment]
Interesting... Will try have a look at it tomorrow. Thanks for reporting.

**Comment by benmccallum** [comment]
Hey @sergeyshaykhullin , was going to have a look at this today. Any reason you closed it?

**Comment by benmccallum** [comment]
I added a `.ToArray()` in v3 onto the rendering of the Failures, as it seems like the serializer wasn't able to recognise the linq projection and render it as an array.

I believe I didn't notice this as I don't use the graphql serializer in any of my tests, I use Verify, which uses a different serializer (Newtonsoft). Fun one :P 

v3 targets FluentValidation v9+, but if you need support for FV v8.* I'll back port this fix too as a v2.0.1.

Hope that helps :)

**Comment by benmccallum** [comment]
Fixed in v2.0.1 and v3
